### PR TITLE
feat: upgrade frontend packages

### PR DESCRIPTION
**Pull request by ljkbennett**

### CODE REVIEW

File: apps/juicebox_web/assets/js/container/container.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  }
+  table {
+    border-collapse: collapse;
+    border-spacing: 0;
+  }
+
+  html {
+    box-sizing: border-box;
+  }
+  *, *:before, *:after {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:54]: Each selector in a comma sequence should be on its own single line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperselector)

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:54]: Each selector in a comma sequence should be on its own single line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperselector)

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:54]: Begin pseudo elements with double colons: `::`

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:54]: Begin pseudo elements with double colons: `::`

File: apps/juicebox_web/assets/js/container/container.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    content: none;
+  }
+  table {
+    border-collapse: collapse;
+    border-spacing: 0;
+  }
+
+  html {
+    box-sizing: border-box;
+  }
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:53]: Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  blockquote, q {
+    quotes: none;
+  }
+  blockquote:before, blockquote:after,
+  q:before, q:after {
+    content: '';
+    content: none;
+  }
+  table {
+    border-collapse: collapse;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:47]: Properties should be ordered border-spacing, border-collapse

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    list-style: none;
+  }
+  blockquote, q {
+    quotes: none;
+  }
+  blockquote:before, blockquote:after,
+  q:before, q:after {
+    content: '';
+    content: none;
+  }
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:45]: Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  ol, ul {
+    list-style: none;
+  }
+  blockquote, q {
+    quotes: none;
+  }
+  blockquote:before, blockquote:after,
+  q:before, q:after {
+    content: '';
+    content: none;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:44]: Property `content` already defined on line 43

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#duplicateproperty)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    line-height: 1;
+  }
+  ol, ul {
+    list-style: none;
+  }
+  blockquote, q {
+    quotes: none;
+  }
+  blockquote:before, blockquote:after,
+  q:before, q:after {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:42]: Each selector in a comma sequence should be on its own single line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperselector)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  body {
+    line-height: 1;
+  }
+  ol, ul {
+    list-style: none;
+  }
+  blockquote, q {
+    quotes: none;
+  }
+  blockquote:before, blockquote:after,
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:41]: Begin pseudo elements with double colons: `::`

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:41]: Begin pseudo elements with double colons: `::`

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:41]: Begin pseudo elements with double colons: `::`

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:41]: Each selector in a comma sequence should be on its own single line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperselector)

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:41]: Begin pseudo elements with double colons: `::`

File: apps/juicebox_web/assets/js/container/container.scss [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  }
+  body {
+    line-height: 1;
+  }
+  ol, ul {
+    list-style: none;
+  }
+  blockquote, q {
+    quotes: none;
+  }
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:40]: Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+  footer, header, hgroup, menu, nav, section {
+    display: block;
+  }
+  body {
+    line-height: 1;
+  }
+  ol, ul {
+    list-style: none;
+  }
+  blockquote, q {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:38]: Each selector in a comma sequence should be on its own single line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperselector)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+  article, aside, details, figcaption, figure,
+  footer, header, hgroup, menu, nav, section {
+    display: block;
+  }
+  body {
+    line-height: 1;
+  }
+  ol, ul {
+    list-style: none;
+  }
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:37]: Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+  /* HTML5 display-role reset for older browsers */
+  article, aside, details, figcaption, figure,
+  footer, header, hgroup, menu, nav, section {
+    display: block;
+  }
+  body {
+    line-height: 1;
+  }
+  ol, ul {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:35]: Each selector in a comma sequence should be on its own single line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperselector)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+    vertical-align: baseline;
+  }
+  /* HTML5 display-role reset for older browsers */
+  article, aside, details, figcaption, figure,
+  footer, header, hgroup, menu, nav, section {
+    display: block;
+  }
+  body {
+    line-height: 1;
+  }
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:34]: Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: apps/juicebox_web/assets/js/container/container.scss [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+  }
+  /* HTML5 display-role reset for older browsers */
+  article, aside, details, figcaption, figure,
+  footer, header, hgroup, menu, nav, section {
+    display: block;
+  }
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/container/container.scss:31]: Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 104 possible new issues (including those that may have been commented here).
* 9 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/66.

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 100 possible new issues (including those that may have been commented here).
* 9 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/66.

File: apps/juicebox_web/assets/js/store.js [hunk 14]
```diff
@@ -0,0 +1,26 @@
+import { createStore as originalCreateStore, applyMiddleware, compose } from 'redux';
+import { createLogger } from 'redux-logger';
+import { routerMiddleware } from 'react-router-redux';
+import thunk from 'redux-thunk';
+import history from './utils/browser-history';
+
+let enhancedCreateStore;
+
+if (__DEV__) {
+  enhancedCreateStore = compose(
```
Comment (OCannings) [inline apps/juicebox_web/assets/js/store.js]: could explore something like this:

```javascript
function devMiddleware(middleware, args = []) {
  if (__DEV__) {
    return middleware.apply(this, args);
  }

  // return a noop
}

applyMiddleware(
  thunk,
  routerMiddleware(history),
  devMiddleware(createLogger)
)
```

**Review by OCannings — APPROVED** [review]

**Comment by ljkbennett** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 9 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/66.

**Pull request merged**